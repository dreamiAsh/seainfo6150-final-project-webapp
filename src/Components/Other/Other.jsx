import React, {useEffect, useState} from "react";
import Styles from "./Other.module.css"

const Other = (props) => {

    return (
        <div className={Styles.box}>
            <div className={Styles.imgBox}>
                <img className={Styles.img} src="https://covid19.lacounty.gov/wp-content/uploads/GettyImages-1128687123-1024x683.jpg" alt="poster image"/>
            </div>
            <h5 className={Styles.title}>{props.title}</h5>
            <p className={Styles.content}>{props.content}</p>
        </div>
    )
}

export default Other
