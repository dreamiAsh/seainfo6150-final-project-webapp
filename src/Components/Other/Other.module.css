.box{
    text-align: center;
}
.imgBox{
    width: 60%;
    height: 280px;
    overflow: hidden;
    display: inline-block;
    margin-top: 60px;
}
.img{
    width: 60%;
}
.title{
    font-size: 22px;
    color: grey;
    margin: 26px 0;
}
.content{
    width: 70%;
    line-height: 28px;
    display: inline-block;
    color: grey;
    margin-bottom: 120px;
}