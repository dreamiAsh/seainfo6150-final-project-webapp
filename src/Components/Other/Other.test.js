import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Other from "./Other.jsx";

it("renders the Other component correctly", () => {
  const { container } = render(<Other />);
  expect(container).toMatchSnapshot();
});