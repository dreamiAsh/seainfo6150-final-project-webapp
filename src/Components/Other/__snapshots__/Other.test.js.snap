// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Other component correctly 1`] = `
<div>
  <div
    class="box"
  >
    <div
      class="imgBox"
    >
      <img
        alt="poster image"
        class="img"
        src="https://covid19.lacounty.gov/wp-content/uploads/GettyImages-1128687123-1024x683.jpg"
      />
    </div>
    <h5
      class="title"
    />
    <p
      class="content"
    />
  </div>
</div>
`;
